utils/s3.py

```diff
@@ -36,3 +36,16 @@ def upload(s3_conn, filename, s3_path):
         name='{}/{}'.format(prefix, filename.replace('/', '_'))
     )
     key.set_contents_from_filename(filename)
+
+def download(s3_conn, out_filename, s3_path):
+    bucket_name, prefix = split_s3_path(s3_path)
+    bucket = s3_conn.get_bucket(bucket_name)
+    key = boto.s3.key.Key(
+        bucket=bucket,
+        name=prefix
+    )
+    logging.warning('loading from %s into %s', key, out_filename)
```

Comment:
I actually made this change, but this line shouldn't be a warning (I had initially made this a warning so it would pass the default log level in my test suite)